### PR TITLE
Stock interface

### PR DESCRIPTION
**Pull request by benhaynes**
Working, but there is a bug with the chart.js tooltip that crahes the chart.

### CODE REVIEW

File: extensions/core/interfaces/stock/interface.vue [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+</script>
+
+<style lang="scss" scoped>
+.v-input {
+  max-width: var(--width-x-small);
+  display: inline-block;
+}
+.v-input input {
+  text-transform: uppercase;
+  // Why can't I style this input?????
```
Comment (rijkvanzanten) [inline extensions/core/interfaces/stock/interface.vue]: Styling is scoped to the current component. Everything inside the `v-input` component is a level deeper and therefore can't be reached from the parent. 

There _is_ a way to override te scope: `.v-input /deep/ input` but I highly recommend not using it, as it defeats the whole purpose of using scoped styles in the first place. Instead, consider adding a prop to the `v-input` component that adds this styles in the component itself.

File: extensions/core/interfaces/stock/interface.vue [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+        this.message = "Please enter a stock symbol...";
+        this.showChart = false;
+        this.latestPrice = "0.00";
+        this.change = "0.00";
+        this.changePercent = "0.00%";
+        this.trending = "trending_flat";
+      }
+    }
+  },
+  created: function () {
```
[the remark this answers is not in the document]
Comment (benhaynes) [reply extensions/core/interfaces/stock/interface.vue]: I don't even know where to adjust this one :/

File: extensions/core/interfaces/stock/interface.vue [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+    this.debouncedGetStock = this.$lodash.debounce(this.getStock, 1000);
```
[the remark this answers is not in the document]
Comment (benhaynes) [reply extensions/core/interfaces/stock/interface.vue]: I thought it might be useful to have both options... immediate or debounced. No?

**Comment by rijkvanzanten** [comment]
This interface was made as a proof of concept and is now too outdated to be merged. We can use code as example

**Pull request closed without merging**